### DISCUSSION TITLE
Dynamic Previews unchecked after restart

### DISCUSSION THREAD

**Discussion by Aisub** [photoprism/photoprism#3864] [Q&A]
![Untitled](https://github.com/photoprism/photoprism/assets/48910143/d5423c1c-010e-40f8-b3dd-14cb5e6434cb)

dynamic previews is turned off after a restart. I have to manually check the check box again to make it work.

**Comment by lastzero** [comment]
It can be that your config folder / file is not writable or that your hardware is not powerful enough for this. Unfortunately, you didn't use our bug report template. Please also make sure to check the server logs for any helpful information before reporting a bug.

**Reply by lastzero** [reply]
@Aisub I tested it and had no problems. My guess is that it could be caused by incorrect filesystem permissions for the `storage/config` folder or for the `storage/config/options.yml` file. Also make sure to check the logs for any helpful information:
- https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3
